src/etc/inc/services.inc

```diff
@@ -529,7 +613,11 @@ function services_dhcpdv4_configure() {
 						} else {
 							$itemtype = "text";
 						}
-						$custoptions .= "option custom-{$dhcpif}-{$poolidx}-{$itemidx} code {$item['number']} = {$itemtype};\n";
+                                                $optionname = "custom-{$dhcpif}-{$poolidx}-{$itemidx}";
+                                                if (array_key_exists($item['type'], $dhcpoptionsname)) {
+                                                	$optionname = $dhcpoptionsname[$item['type']];
+                                                }
+						$custoptions .= "option $optionname code {$item['number']} = {$itemtype};\n";
```

Comment:
This change is not correct. This is the place where custom options are defined and not where value is attributed. Then in this place you should just skip $custoptions definition if it's a known item. Other than that, you should note there is a similar block above at line 598 that must be changed as well.

The block where value is attributed you will find at line 912 and this is where you will use values defined in $dhcpoptionsname.

As a general note, please keep indent rules using TABs with size of 8